etcdserver/apply.go

```diff
@@ -143,6 +144,8 @@ func (a *applierV3backend) Apply(r *pb.InternalRaftRequest) *applyResult {
 		ar.resp, ar.err = a.s.applyV3.AuthEnable()
 	case r.AuthDisable != nil:
 		ar.resp, ar.err = a.s.applyV3.AuthDisable()
+	case r.AuthStatus != nil:
+		ar.resp, ar.err = a.s.applyV3.AuthStatus()
```

Comment:
My understanding is that we add AuthStatus only for compatibility, not back-porting full feature. https://github.com/etcd-io/etcd/pull/11536/files#diff-583048be54c04f9555a60191ec8767757fc7f0c477d5bc7f2191af8a3c1bf920

In this case, maybe it's simpler to `return nil` here and not backport applyV3.AuthStatus() and AuthStatusResponse.

Reply:
This can be optional. But I think having the real response is
1. better for readability. 
2. 3.4 does not really know what the response could be used for in 3.5. It is probably safer to return the real response.

Reply:
unless you add [EtcdServer.AuthStatus ](https://github.com/etcd-io/etcd/pull/11536/files#diff-583048be54c04f9555a60191ec8767757fc7f0c477d5bc7f2191af8a3c1bf920) that registers request, response won't be used anywhere. This [Trigger](https://github.com/etcd-io/etcd/blob/release-3.4/etcdserver/server.go#L2275) in apply will be noop.

Reply:
set to nil